src/redux/actions/dictionaries/dictionaryActionCreators.js

```diff
@@ -111,6 +112,24 @@ export const fetchDictionaryConcepts = data => (dispatch) => {
         dispatch(isFetching(false));
       });
 };
+
+export const releaseHead = (url, data) => (dispatch) => {
+  return api.dictionaries
+    .realisingHeadVersion(url, data)
+    .then(
+      (payload) => {
+        dispatch(realisingHeadSuccess(payload));
+        dispatch(isFetching(false));
+        notify.show(
+          'Head Version Successfully Released',
+          'success', 6000,
+        );
+      }
+    )
+    .catch(() => {
+          notify.show("Network Error", 'error', 6000);
```

Comment:
Assuming `dispatch(isFetching(false))` stops the loader or the likes. Shouldn't you be doing the same here.

Reply:
Thanks for the observation. I had missed that.